### ISSUE TITLE
[feature]: UI and usability improvements

### ISSUE THREAD

**Issue by mbombonato** [hoppscotch/hoppscotch#2574]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

First, thanks for your work on this nice app. My team is migrating from other similar tools and we have a few suggestions to improve our usability:
- Keep a local cached version of Team Collections (sometimes is very slow to load everything)
- Save the last state when close the app (keep the last request done, keep the collection tree opened and so on...)
- Move the environment button to the main screen (similar to Postman and Insomnia). Different colors for each ENV would be a nice plus.
![Screenshot from 2022-08-10 13-37-34](https://user-images.githubusercontent.com/38539268/183966614-afde3336-dd46-4594-9fcd-881a5704c469.png)

### Why should this be worked on?

It will improve a lot our usability

**Comment by AndrewBastin** [comment]
Thank you for your enthusiasm @mbombonato for Hoppscotch! <3

But, please do note that one issue is meant for one issue/feature request/bug etc. Please split this issue off for each of the concerns mentioned.

**Comment by furier** [comment]
Yes I find the environment selection very hidden and cumbersome now
